GovUkDesignSystem/Attributes/ValidationAttributes/GovUkValidateCustom.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUkDesignSystem.Attributes.ValidationAttributes;
+
+[AttributeUsage(AttributeTargets.Property)]
+public class GovUkValidateCustom: ValidationAttribute
```

Comment:
```suggestion
public class GovUkValidateCustom : ValidationAttribute
```
Does this need formatting, or does the formatter accept this as-is?

Reply:
Also wonder if the name is right. Reads a bit awkwardly. At the very least it feels like 'custom' is an adjective so should come before 'validation'. But also I think these are usually named to be adjectival, as they describe the property: e.g. `Required`. Perhaps `GovUkHasCustomValidator`?